src/Microsoft.Diagnostics.Monitoring.EventPipe/Configuration/MetricSourceConfiguration.cs

```diff
@@ -45,6 +46,37 @@ public MetricSourceConfiguration(float metricIntervalSeconds, IEnumerable<string
                })).ToList();
         }
 
+        public MetricSourceConfiguration(float metricIntervalSeconds, IEnumerable<string> customProviderNames, int maxHistograms, int maxTimeSeries) : this(metricIntervalSeconds, customProviderNames)
+        {
+            const long TimeSeriesValues = 0x2;
```

Comment:
Consider allowing _just_ eventcounters or _just_ Meters, instead of always forcing both.